src/it/projects/GH-502_zipDirPrefix/src/main/java/module-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+module com.corporate.project {
+}
```

Comment:
Probably doesn't matter, but maybe we can find another example module

Reply:
Can you explain details of your comment?

This testcase creates launcher as following:
```xml
<launcher>helloworld=com.corporate.project/com.corporate.project.HelloWorld</launcher>
```

I think the app should be modulalized for `--launcher` option in jlink. So I think `module-info.java` is necessary for this test.